src/topics/yaml-guide.md

```diff
@@ -25,7 +25,7 @@ You can skip this section if you are already comfortable with YAML.
 
 Fundamentally, a file written in YAML consists of a set of _key-value pairs_.
 Each pair is written as `key: value`,
-where whitespace after the `:` is optional.
+where whitespace after the `:` is optional, but is required when defining fields in a CWl file.
```

Comment:
I think this change made its way into your other PR :+1: same comment: s/CWl/CWL

Reply:
That's true. I didn't notice. Is it okay now?

Reply:
I think you need a capital L in CWL